syntax = "proto3";

import "library.proto";

package espy;

service Espy {
    rpc GetLibrary(LibraryRequest) returns (LibraryResponse) {}
  }
  
  message LibraryRequest {
    string user_id = 1;
  }
  
  message LibraryResponse {
    Library library = 1;
  }
  